bookkeeping/bookkeeper.go

```diff
@@ -0,0 +1,123 @@
+package bookkeeping
+
+import (
+	"bytes"
+	"github.com/Comcast/webpa-common/logging"
+	"github.com/go-kit/kit/log"
+	"net/http"
+	"time"
+)
+
+type CapturedResponse struct {
+	Code    int
+	Payload []byte
+	Header  http.Header
+}
+
+// RequestFunc takes the request and returns key value pairs from the request
+type RequestFunc func(request *http.Request) []interface{}
+
+// ResponseFunc takes the ResponseWriter and returns key value pairs from the request
+type ResponseFunc func(response CapturedResponse) []interface{}
+
+// Handler is for logging of the request and response
+type Handler struct {
```

Comment:
This shouldn't be exported from the package, given that you have a `New` constructor.  This prevents a lot of confusion over how the package's code should be used.